go/kbnm/host_json.template

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "io.keybase.kbnm",
+  "description": "Keybase Native Messaging API",
+  "path": "@@HOST_PATH@@",
```

Comment:
I picked this style pretty much at random for the installer, so let me know if you don't like it :)